src/main/resources/application.properties

```diff
@@ -1,5 +1,12 @@
-db.user=conduit
-db.pass=*$X1ROzcs%fg
+#db.user=conduit
+#db.pass=*$X1ROzcs%fg
+#db.driver=org.mariadb.jdbc.Driver
+#db.database=conduit
+#db.host=jdbc:mysql://localhost
+
+db.user=root
```

Comment:
Hmm I was thinking it would be easiest to make a non root user for the app. Otherwise people need to change their root password for the server to run. I also saw you used H2. maybe we should migrate to H2 to make the setup easier.